tests/exemplars.ttl

```diff
@@ -28,6 +29,11 @@ kb:Action-13c91d40-9ca0-4558-81a1-4f7db888c371
 	gufo:hasEndPointInXSDDateTimeStamp "2021-02-03T04:05:06Z"^^xsd:dateTimeStamp ;
 	.
 
+kb:Bearing-74c2bb20-1e4f-40ac-807e-bc2c528e39da
+	a drafting:Bearing ;
+	gufo:inheresIn kb:Vehicle-c7e910e8-4bed-473e-ad74-3df3eb63cbcc ;
+	.
```

Comment:
Now, the particular individual, the measured bearing `kb:Bearing-74c2bb....`, has been given a value.

However, you want to express that the bearing is only temporary, so you introduce the Situation at line 92. That's fine, but then there's no need to instantiate individual `kb:Bearing-74c2bb....` anymore. 

So either use the pattern as defined here, or the one that is defined on line 94 and further, but there's no reason to instantiate both, isn't it?

Reply:
My understanding of `QualityValueAttributionSituation` is that the quality-**value** is temporary, but the quality itself could persist outside of the time the situation holds.

I admit I don't understand why the `QualityValueAttributionSituation` does not reference the quality instance, and instead references the quality type.  Maybe it's for applications where the quality-type is semi-rigid, and each continuous interval of time where the quality of exists is a different instance of the quality-type?

Reply:
Thank you @plbt5 for the correction in discussion: The situation where the quality-value is held is temporary.  The quality-value is (in the case of bearing and velocity) atemporal/time-independent.

Reply:
As for why `QualityValueAttributionSituation` describes the quality type, rather than quality instances: consider situations where two individuals are compared along the same quality type.  To express a situation where one car is going faster than another, the two cars' `:Velocity` instances would need to share the type linked by `concernsQualityType`.

I think my supplementary questions are answered well enough.

Reply:
Back to the original question about whether the instantiated quality should be here if it's not used:

The reason for instantiating the quality, and not necessarily using it in the "abox," is to see if its instantiation triggers any errors from the SHACL shapes.  E.g. I've found from this that some classes defined as disjoint-unions are insufficiently specified, and the SHACL shapes informed me so.

While the quality instances in this exemplars graph might not look like they're doing much, I plan to use the same IRIs in another profile that will exercise quality-value assignment, and then see if the SHACL shapes of both profiles consider there to be any inconsistencies when it's all brought together in [CASE-Examples-QC](https://github.com/ajnelson-nist/CASE-Examples-QC).